__tests__/app.test.js

```diff
@@ -326,3 +326,26 @@ describe("PATCH /api/reviews/:review_id (votes)", () => {
       });
   });
 });
+describe("DELETE /api/comments/:comment_id", () => {
+  test("204: should return no content when passed the correct comment_id", () => {
+    return request(app).delete("/api/comments/6").expect(204);
```

Comment:
We could extend this test a little further and assert that we expect the response body to be empty

Reply:
np, just hoped the status code 204 was enough to show no content but i'll be more explicit :)